docker-compose.services.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.9'
+
+services:
+  calmpal_db:
+    container_name: calmpal_db
+    image: postgres:15.2
+    environment:
+      - POSTGRES_DB=$(echo "$DB_CONNECTION_STRING" | awk -F[@/:] '{print $8}')
+      - POSTGRES_USER=$(echo "$DB_CONNECTION_STRING" | awk -F[@/:] '{print $4}')
+      - POSTGRES_PASSWORD=$(echo "$DB_CONNECTION_STRING" | awk -F[@/:] '{print $5}')
+    ports:
+      - 5434:5432
```

Comment:
Will this work if we have a different connection string in .env?

Reply:
Good point) Will change into .env.example

Reply:
I had postgres locally with 5432 port, so to prevent ambiguous port mapping